changelog.d/11398.feature

```diff
@@ -0,0 +1 @@
+Implement MSC3383 for including the destination in S2S authentication headers.
```

Comment:
```suggestion
Implement [MSC3383](https://github.com/matrix-org/matrix-doc/pull/3383) for including the destination in server-to-server authentication headers.
```

You can include "Contributed by your name." (or Famedly) if you would like, by the way.

Reply:
Is it good like this?

Reply:
Looks great! Thanks!